.changeset/good-swans-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@paprika/popover": minor
+---
+
+When popover opened pressing ESC key only closes popover content and stops propagation. Benefical if using the popover in a Panel or TakeOver component
```

Comment:
Would be good to include a Storybook story to illustrate a Popover in a Modal/Takeover/Panel to verify this desired behaviour.

Reply:
There is this example http://storybooks.highbond-s3.com/paprika/master/?path=/story/messaging-takeover-backyard-sandbox--with-nested-popover. I'll add one for the Panel as well

Reply:
http://storybooks.highbond-s3.com/paprika/master/?path=/story/messaging-takeover-backyard-sandbox--with-nested-popover

Went to that example, I'm clicking `esc` after opening the popover should I expect only the popover to close? because right now is closing even the takeover.

I even open the popover click tab, focus on the link, click `esc`, and still close popover plus takeover.

Is this expected?

Reply:
That `s3` link seems stale. I'm trying it out locally and it seems okay.